tests/foreman/api/test_computeresource_gce.py

```diff
@@ -204,18 +205,28 @@ def test_positive_crud_gce_cr(self, module_org, module_location, gce_cert):
         assert not entities.GCEComputeResource().search(query={'search': f'name={new_name}'})
 
     @pytest.mark.tier3
-    def test_positive_check_available_images(self, module_gce_compute, googleclient):
-        """Verify all the images from GCE are available to select from
+    def test_positive_check_available_images(self, module_gce_compute, googleclient, gce_cert):
```

Comment:
```suggestion
    def test_positive_check_available_images(self, module_gce_compute, googleclient, gce_cert):
```
Is `gce_ cert` needed here?

Reply:
Updated!